tests/platform_tests/cli/test_show_chassis_module.py

```diff
@@ -0,0 +1,82 @@
+import logging
+import pytest
+from tests.common.helpers.assertions import pytest_assert
+from util import get_field_range, get_fields, get_skip_mod_list
+
+
+pytestmark = [
+    pytest.mark.topology('t2')
+]
+
+CMD_SHOW_CHASSIS_MODULE = "show chassis-module"
+
+
+def parse_chassis_module(output, expected_headers):
+    assert len(output) > 2
+    f_ranges = get_field_range(output[1])
+    headers = get_fields(output[0], f_ranges)
+
+    for header_v in expected_headers:
+        pytest_assert(header_v in headers, "Missing header {}".format(header_v))
+
+    result = {}
+    for a_line in output[2:]:
+        field_val = get_fields(a_line, f_ranges)
+        mod_idx = field_val[0]
+        result[mod_idx] = {}
+        cur_field = 1
+        for a_header in headers[1:]:
+            result[mod_idx][a_header] = field_val[cur_field]
+            cur_field += 1
+
+    return result
+
+
+def test_show_chassis_module_status(duthosts, enum_dut_hostname):
+    cmd = " ".join([CMD_SHOW_CHASSIS_MODULE, "status"])
+    logging.info("verifying output of cli command {}".format(cmd))
```

Comment:
It would be better to initialize a logger than directly using the logging module.

Reply:
@wangxin 
I made changes and committed them as per your comment